.gitlab-ci.yml

```
@@ -210,3 +210,4 @@ artifact:
     - ./deploy/build
     - ./deploy/build-post-config
     - make build_artifact ARTIFACT_DESTINATION_FILE='./tmp/idp.tar.gz'
+    - bundle exec ./scripts/artifact-upload './tmp/idp.tar.gz'
```

could we move `ARTIFACT_DESTINATION_FILE` to the variables block above and reuse it on both these lines?

```suggestion
    - make build_artifact
    - bundle exec ./scripts/artifact-upload $ARTIFACT_DESTINATION_FILE
```